pysatNASA/instruments/methods/mgs.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# TODO(#XXX): Update NRL review before next version release
```

Comment:
```suggestion
# TODO(#249): Update NRL review before next version release
```

Reply:
I think you can remove this comment for now.